src/sinks/aws_cloudwatch_logs/retry.rs

```diff
@@ -32,27 +32,31 @@ impl<T: Send + Sync + 'static> RetryLogic for CloudwatchRetryLogic<T> {
     type Error = CloudwatchError;
     type Response = T;
 
+    // TODO this match may not be necessary given the logic in `is_retriable_error()`
```

Comment:
Something to look into later?

Reply:
Likely, originally I deemed it outside of the scope of the dependency upgrades - at worst it's just repeated logic.